### PR TITLE
chore(debug): set envs in each debug handler

### PR DESCRIPTION
**Pull request by kuojianlu**

### CODE REVIEW

File: packages/fx-core/src/component/debugHandler/localEnvProvider.ts [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+      SslKeyFile: "SSL_KEY_FILE",
+    },
+    teamsfx: {
+      ClientId: "REACT_APP_CLIENT_ID",
+      LoginUrl: "REACT_APP_START_LOGIN_PAGE_URL",
+      FuncEndpoint: "REACT_APP_FUNC_ENDPOINT",
+      FuncName: "REACT_APP_FUNC_NAME",
+    },
+  },
+  backend: {
```
Comment (swatDong) [inline packages/fx-core/src/component/debugHandler/localEnvProvider.ts:34]: How to handle those place-holder envs like `SQL_xxx`?

Comment (kuojianlu) [reply packages/fx-core/src/component/debugHandler/localEnvProvider.ts:34]: Should we put empty SQL envs in the file, or totally let user to manage the SQL envs?

Comment (swatDong) [reply packages/fx-core/src/component/debugHandler/localEnvProvider.ts:34]: Better to have those when first time generating the file. We do let user to manage the values, but need place holders for keys.

Comment (kuojianlu) [reply packages/fx-core/src/component/debugHandler/localEnvProvider.ts:34]: Added SQL env place holders when the project has SQL.

**Review by swatDong — APPROVED** [review]

**Pull request merged**